src/game.js

```diff
@@ -0,0 +1,115 @@
+const _ = require('lodash')
+const Betting = require('./betting')
+const currency = require('./currency')
+
+module.exports = class Game {
+  constructor() {
+    this.minBet = 10;
```

Comment:
The terminating semicolon is inconsistently used in your code; I prefer omitting them entirely, but please be consistent.